pkg/peerhealth/server.go

```diff
@@ -111,7 +101,8 @@ func (s *Server) Start(ctx context.Context) error {
 }
 
 // IsHealthy checks if the given node is healthy
-func (s Server) IsHealthy(ctx context.Context, request *HealthRequest) (*HealthResponse, error) {
+func (s *Server) IsHealthy(ctx context.Context, request *HealthRequest) (*HealthResponse, error) {
```

Comment:
I made this change in my test PR, but I don't think it is really needed anymore

Reply:
you mean the pointer receiver? I made it today because some receivers were pointers, some not

Reply:
Ah ok, so just a coincidence :grin: :+1: